UnityProject/Assets/Scripts/Objects/Doors/ConstructibleDoor.cs

```diff
@@ -102,6 +105,14 @@ public void ServerPerformInteraction(HandApply interaction)
 					() => WhenDestroyed(null));
 				}
 			}
+			if (Validations.HasUsedComponent<AirlockPainter>(interaction))
+			{
+				AirlockPainter painter = interaction.HandObject.GetComponent<AirlockPainter>();
+				if (painter)
+				{
+					painter.ChoosePainJob(gameObject);
```

Comment:
This is being called in ServerPerformInteraction, you'll need to call it in WillInteract instead

Reply:
Many thanks. Indeed, the problem was that I was calling the radial menu in ServerPerformInteraction.